apps/hashdotai/blog/008_vaccine-distribution.mdx

```diff
@@ -22,11 +22,11 @@ We’ve constructed a HASH simulation to help understand these complexities. You
 
 This [Vaccine Supply Coordination model](https://hash.ai/@hash/vaccination-supply-coordination) is an extension of our [Flexible Supply Chain model](https://hash.ai/@hash/flexible-supply-chain). It simulates the distribution of a vaccine on an hourly scale to a community through three different types of agents: distribution centers, medical facilities, and a vaccine source.
 
-![](images/sim.png)
+{/* ![](images/sim.png) */}
```

Comment:
@nonparibus I've commented this out because the image is missing, causing MDX compilation to break for this page

Reply:
Sorry, had missed that one. Updated in https://github.com/hashintel/hash/pull/2069/commits/97c116de59ce7419541bbf06510f478bbdabcea3